fed/proxy/barriers.py

```diff
@@ -17,6 +17,7 @@
 import threading
 import time
 import copy
+import importlib.metadata
```

Comment:
Turns out `importlib.metadata` is a standard library in Python 3.8 and later, making it incompatible for py37

Reply:
This [PR](https://github.com/ray-project/rayfed/pull/155) resolves a bug caused by Python version compatibility.